openwisp_users/settings.py

```diff
@@ -1,8 +1,8 @@
 from django.conf import settings
 from openwisp_utils.utils import default_or_test
 
-ORGANIZATON_USER_ADMIN = getattr(settings, 'OPENWISP_ORGANIZATON_USER_ADMIN', False)
-ORGANIZATON_OWNER_ADMIN = getattr(settings, 'OPENWISP_ORGANIZATON_OWNER_ADMIN', True)
+ORGANIZATION_USER_ADMIN = getattr(settings, 'OPENWISP_ORGANIZATION_USER_ADMIN', False)
```

Comment:
unfortunately we need to be backward compatible with the mistaken setting, otherwise existing applications will change inadvertently. 

something like this should work:

```python
ORGANIZATION_USER_ADMIN = (
    # setting with typographic error maintained for backward compatibility
    # TODO: remove in openwisp-users 0.5
    getattr(settings, 'OPENWISP_ORGANIZATON_USER_ADMIN', False) or
    getattr(settings, 'OPENWISP_ORGANIZATION_USER_ADMIN', False)
)
```

We'll then create an issue to remind us to remove this in the next point version.

Reply:
Okay, will do that. Learned something new today :smile:

Reply:
@nemesisdesign Changing 'ORGANIZATION' back to 'ORGANIZATON' results in test failure. Maybe we should keep it on hold until a new version is released.